### DISCUSSION TITLE
Display Chinese char problem

### DISCUSSION THREAD

**Discussion by michaelbr51** [tsujan/FeatherPad#497] [General]
How to display Chinese char in Featherpad?
I have a subtitle in Chinese, whey I tried to display it in Featherpad it showed garbages, but Aegisub showed OK, I'm using MX Linux 19.3 and Featherpad 0.16.0
[Assembly.gb.srt.txt](https://github.com/tsujan/FeatherPad/files/6163513/Assembly.gb.srt.txt)
, I've attached the subtitle if needed.
Assembly.gb.srt.txt
ps: 1) I think my Linux is configured properly, I can see Chinese char in Brave browser. I've also installed Chinese char package in Linux. It should be a configuration problem but can't find it.
2) I changed the file name from Assembly.gb.srt to Assembly.gb.srt.txt, so I can add it to this comment.

Thanks

**Comment by tsujan** [comment]
Change the encoding in Options → Encoding. Usually, if you have a Chinese locale, that won't be needed but it's there for such cases:

![chinese](https://user-images.githubusercontent.com/981076/111618526-b9dff200-87f9-11eb-860a-b0b94e66d323.png)

Qt6 doesn't support legacy encodings anymore. Because of that, when I port FeatherPad to Qt6, I'll have to remove the encoding options. [As Qt6 developers say](https://lists.qt-project.org/pipermail/development/2020-May/039455.html), legacy encodings "should not be there anymore in 2020." I agree with them.

**Comment by tsujan** [comment]
BTW, FeatherPad 0.16.0 is outdated, although the latest version (0.18.0) doesn't make a difference in this regard. No app can guess all encodings correctly.

**Reply by michaelbr51** [reply]
Thanks so much for your prompt reply and tips.

**Reply by tsujan** [reply]
You're welcome!
